runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/SplittableParDoNaiveBounded.java

```diff
@@ -134,9 +134,30 @@ public PCollectionTuple expand(PCollection<KV<byte[], KV<InputT, RestrictionT>>>
     }
 
     @Setup
-    public void setup() {
+    public void setup(SetupContext c) {
```

Comment:
Other `DoFn` methods support a `PipelineOptions` type argument and the `DoFnSignature` detects this.

Reply:
Removed SetupConext and use PipelineOptions directly.